GettingStarted.md

```diff
@@ -0,0 +1,162 @@
+# Introduction
+Welcome to impress.js! This presentation framework allows you to create stunning presentations with the power of CSS3 transformations. It is inspired by the idea behind [prezi.com](https://prezi.com). 
+**NOTE:** This Guide is not made for you, if you have never written HTML and/or CSS before. Knowing your way around in JavaScript certainly helps, but is not a necessity. You may still continue this tutorial and try to understand what we do as you go. 
+
+Also, if you are an experienced (or sorta experienced user, anyway), you might prefer the [DOCUMENTATION](DOCUMENTATION.md).
```

Comment:
I'm sorry to be a language nitpick...This kind of casual language can be nice in a way, but also it makes the text longer without adding information value. I would prefer: 

    For more advanced and complete documentation, see [DOCUMENTATION](DOCUMENTATION.md)